fleetspeak/src/client/daemonservice/execution/execution.go

```diff
@@ -419,20 +419,29 @@ func (e *Execution) readMsg() *fspb.Message {
 func (e *Execution) statsLoop() {
 	defer e.inProcess.Done()
 	pid := e.cmd.Process.Pid
+	var version string
 	select {
 	case sd := <-e.startupData:
 		if int(sd.Pid) != pid {
 			log.Infof("%s's self-reported PID (%d) is different from that of the process launched by Fleetspeak (%d)", e.daemonServiceName, sd.Pid, pid)
 			pid = int(sd.Pid)
+			version = sd.Version
```

Comment:
We still want the version even if the pids match.

Reply:
done